bpm/bpm_extractor.py

```diff
@@ -8,15 +9,16 @@
 
 
 def get_song_id(filename):
-    return filename.split("/")[-1].split("-")[0]
+    return re.search(r"([0-9]+-[0-9]+-[0-9]+)", filename).group(0)
```

Comment:
Possibly dangerous if song id is not in name, but should be fine

Reply:
I think we have to assume the data from DR is sound. We were told all data from 'Diskoteket' would be a certain way. I don't see how we could recreate the ID if we did not get it.